### ISSUE TITLE
[FEAT] Save scroll location when navigating back

### ISSUE THREAD

**Issue by mrtumnus** [AnalogJ/scrutiny#91]
**Is your feature request related to a problem? Please describe.**
When I click on a hard drive to view S.M.A.R.T. attributes and navigate Back to the main menu, the browser returns to the top of the page.

**Describe the solution you'd like**
The last scroll location should be saved when navigating to a drive detail view.

**Additional context**

**Comment by AnalogJ** [comment]
Yeah, this would be a nice feature for uses with lots of drives. Good idea 👍

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.
